assets/js/modules/analytics/components/dashboard/DashboardUniqueVisitorsWidget.stories.js

```diff
@@ -0,0 +1,215 @@
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+	},
+};
+
+export const LoadingEntityURL = WidgetAreaTemplate.bind();
+LoadingEntityURL.storyName = 'Loading with entity URL set';
+LoadingEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsCompareEntityURL ), { options: optionsCompareEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( getAnalyticsMockResponse( optionsEntityURL ), { options: optionsEntityURL } );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).startResolution( 'getReport', [ optionsCompareEntityURL ] );
+	},
+};
+
+export const DataUnavailableEntityURL = WidgetAreaTemplate.bind();
+DataUnavailableEntityURL.storyName = 'Data Unavailable with entity URL set';
+DataUnavailableEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveGetReport( [], { options: optionsEntityURL } );
+	},
+};
+
+export const ErrorEntityURL = WidgetAreaTemplate.bind();
+ErrorEntityURL.storyName = 'Error with entity URL set';
+ErrorEntityURL.args = {
+	setupRegistry: ( registry ) => {
+		const { dispatch } = registry;
+		const error = {
+			code: 'missing_required_param',
+			message: 'Request parameter is empty: metrics.',
+			data: {},
+		};
+
+		provideSiteInfo( registry, { currentEntityURL } );
+		dispatch( STORE_NAME ).receiveError( error, 'getReport', [ optionsEntityURL ] );
+		dispatch( STORE_NAME ).finishResolution( 'getReport', [ optionsEntityURL ] );
+	},
+};
+
+export default {
+	title: 'Modules/Analytics/Widgets/DashboardUniqueVisitorsWidget',
+	decorators: [
+		( Story ) => {
+			const setupRegistry = ( registry ) => {
+				const { dispatch } = registry;
+
+				const [ property ] = accountsPropertiesProfiles.properties;
+				registry.dispatch( STORE_NAME ).receiveGetSettings( {
+					// eslint-disable-next-line sitekit/acronym-case
+					accountID: property.accountId,
+					// eslint-disable-next-line sitekit/acronym-case
+					internalWebPropertyID: property.internalWebPropertyId,
+					// eslint-disable-next-line sitekit/acronym-case
+					profileID: property.defaultProfileId,
+				} );
+
+				dispatch( CORE_WIDGETS ).registerWidgetArea( areaName, {
+					title: 'Overview',
+					style: WIDGET_AREA_STYLES.BOXES,
+				} );
+				dispatch( CORE_WIDGETS ).registerWidget( widgetSlug, {
+					Component: DashboardUniqueVisitorsWidget,
+					width: WIDGET_WIDTHS.FULL,
+				} );
+				dispatch( CORE_WIDGETS ).assignWidget( widgetSlug, areaName );
```

Comment:
This part looks like it will be a common need for most widget stories. I'm wondering why we use a widget area here at all rather than just rendering the widget component directly 🤔 

If we really need it, this part (along with maybe the Template component & registration) would be great to extract into something more reusable (maybe from the widget area stories themselves?).

Reply:
@aaemnnosttv You're right. I've updated the stories to render the widget component directly.